packages/web-react/src/components/TooltipModern/demo/TooltipWithFloatingUI.tsx

```diff
@@ -129,6 +129,8 @@ const TooltipWithFloatingUI = () => {
             flipFallbackPlacements={suggestedFallbackPlacement}
             enableShifting={shift}
             enableSizing={size}
+            trigger={['hover']}
```

Comment:
What does it mean `isOpenOnHover` when the `trigger` is set to `hover`. I think that the case is that the Tooltip is clickable when it is open on hover. Shouldn't we this call `isClickableOnHover`? Or does it mean that it is focusable? E.g. `isFocusableOnHover`?

Reply:
i like the name `isFocusableOnHover`, so i changed it to this.